lookout/style/format/utils.py

```diff
@@ -149,3 +151,30 @@ def generate_comment(filename: str, confidence: int, line: int, text: str) -> Co
     comment.line = line
     comment.text = text
     return comment
+
+
+def get_rules_classification_report(y_pred: numpy.ndarray, y_true: numpy.ndarray,
```

Comment:
We don seem to depend on the rules here (do we?) so I suggest to remove "rules" from the name